tests/books/toc/_toc_urllink.yml

```diff
@@ -14,5 +13,5 @@ sections:
   sections:
   - file: subfolder/asubpage
     title: Asubpage
-  - url: https://jupyterbook.org/file-types/markdown.html
+  - url: www.jupyterbook.org/file-types/markdown.html
```

Comment:
@AakashGfude what is driving this change?

Reply:
@mmcky just to test out a faulty URL. 
In case of HTML, if a url does not have `http` or `https` , it is not included in the toc. The same has been done for `tableofocontents` directive. So, the above change was testing out this case.